app/models/site_config.rb

```diff
@@ -55,6 +55,8 @@ class SiteConfig < RailsSettings::Base
   # Community Content
   field :community_name, type: :string, default: ApplicationConfig["COMMUNITY_NAME"] || "New Forem"
   field :community_emoji, type: :string, default: "🌱"
+  field :collective_noun, type: :string, default: "Community"
+  field :collective_noun_disabled, type: :boolean, default: false
```

Comment:
We might want to add a comment next to these 2 fields explaining that we don't use these across the app elsewhere right now and that they are only here temporarily/will be removed when this issue is fully resolved

Reply:
Great call! Done ✔️